clean_project/setup.py

```diff
@@ -0,0 +1,28 @@
+from distutils.core import setup
+
+setup(
+    name="clean_project",
+
+    version="0.1.0",
+
+    author="Gianluca Parente",
+    author_email="gianluca.parente7@gmail.com",
+
+    packages=["project", "project/config"],
+
+    include_package_data=True,
+
+    url="",
+
+    license="LICENSE.txt",
```

Comment:
you don't have any LICENSE.txt file in your folder, so either you add it with the proper content, or remove this line